### PR TITLE
using getpass to prompt the input process

### PR DESCRIPTION
**Pull request by zhangzcjames**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,6 +16,6 @@ def readme():
     author_email="support@nuvolos.cloud",
     license="MIT",
     packages=["nuvolos"],
-    install_requires=["pyodbc"],
+    install_requires=["pyodbc", "keyring"],
```
Comment (andrewsali) [inline setup.py]: getpass is missing?

**Review by andrewsali — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (andrewsali) [reply setup.py]: Also, please respect lexicogrpahic ordering

Comment (zhangzcjames) [reply setup.py]: getpass is from python itself, so we don't need to add it, right?

Comment (andrewsali) [reply setup.py]: ah ok, thx

**Pull request merged**